test/clj/chronograph/domain/user_test.clj

```diff
@@ -61,3 +62,25 @@
                        :created-at         (time/now)
                        :updated-at         (time/now)}
                retrieved-user))))))
+
+(deftest organizations-test
+  (testing "Returns the list of organizations a user belongs to"
+    (let [user (factories/create-user)
+          organization1 (factories/create-organization (:users/id user))
+          organization2 (factories/create-organization (:users/id user))]
+      (with-transaction [tx db/datasource]
+        (let [user-organizations (user/organizations tx user)]
+          (is (= 2 (count user-organizations)))
+          (is (= #{(:organizations/id organization1)
+                   (:organizations/id organization2)}
+                 (set (map :organizations/id user-organizations))))))))
+  (testing "Does not return organizations to which the user does not belong"
+    (let [user (factories/create-user)
+          other-user (factories/create-user)
+          organization1 (factories/create-organization (:users/id user))
+          organization2 (factories/create-organization (:users/id other-user))]
+      (with-transaction [tx db/datasource]
+        (let [user-organizations (user/organizations tx user)]
+          (is (= 1 (count user-organizations)))
+          (is (= #{(:organizations/id organization1)}
+                 (set (map :organizations/id user-organizations)))))))))
```

Comment:
same observation as this comment: https://github.com/nilenso/chronograph/pull/11/files#r498327838